lib/MusicBrainz/Server/Entity/URL/OffizielleCharts.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Entity::URL::OffizielleCharts;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'Offizielle Deutsche Charts' }
```

Comment:
Intentionally left untranslated at the moment since it seems to be the official name of the site, but open to discussion.

Reply:
:ok_hand: Just like we already do for most other websites that do not have official translations/transliterations.